packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -162,3 +162,15 @@ export async function deletePatientList(cohortUuid: string, ac = new AbortContro
     ac,
   );
 }
+
+export async function getPatientListName(patientListUuid: string, ac = new AbortController()) {
+  try {
+    const url = `${cohortUrl}/cohort/${patientListUuid}?`;
+    const { data } = await openmrsFetch<OpenmrsCohort>(url, {
+      signal: ac.signal,
+    });
+    return data.name;
+  } catch (error) {
+    throw error;
+  }
+}
```

Comment:
```suggestion
export async function getPatientListName(patientListUuid: string) {
  const abortController = new AbortController();
    
  try {
    const url = `${cohortUrl}/cohort/${patientListUuid}?`;
    const { data } = await openmrsFetch<OpenmrsCohort>(url, {
      signal: abortController.signal,
    });
    
    return data?.name;
  } catch (error) {
    console.error('Error resolving patient list name: ', error);
  }
}
```

Reply:
A couple points:

- No need to bring in the AbortController from the function call. It can be declared right within the function. 
- If an error occurs while resolving the cohort name, we don't want to render this error in the breadcrumb. We want to log the error message to the console and render a sensible fallback or error state instead. More on this below.